packages/rfw/lib/src/flutter/runtime.dart

```diff
@@ -226,6 +233,11 @@ class Runtime extends ChangeNotifier {
     _clearCache();
   }
 
+  /// Returns an unmodifiable map from LibraryName to WidgetLibrary.
```

Comment:
same as above, docs should be more detailed

Reply:
Done.